packages/jitm/src/Manager.php

```diff
@@ -40,10 +49,13 @@ static function init() {
 	/**
 	 * Jetpack_JITM constructor.
 	 */
-	private function __construct() {
-		if ( ! Jetpack::is_active() || Jetpack::is_development_mode() ) {
+	public function __construct() {
+		/*
```

Comment:
Keep referring to global static methods until the other methods are available. It's too easy to leave this broken otherwise.